### PR TITLE
EDIT: CSS overflow-y

### LINKED ISSUE #4414: [Edit] CSS [Edit] CSS overflow: overflow-y
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like an edit to a term within CSS `overflow-y`. The term entry lives in `content/css/concepts/overflow/terms/overflow-y/overflow-y.md`

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

I added an image that reflects the three examples given in the article. I also made one small grammatical correction.

### Issue Solved

Closes #4414 

### Type of Change

- Editing an existing entry (add image)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by avdhoottt — APPROVED** [review]
Nice image!! Looks good for the second review.

**Review by helfese — APPROVED** [review]

File: content/css/concepts/overflow/terms/overflow-y/overflow-y.md [hunk 1]
```diff
@@ -50,10 +50,14 @@ Provide a scrollbar and clip any content that overflows.
 
 ## Example 3
 
-Provide a scrollbar only if overflow contents exists.
+Provide a scrollbar only if overflow contents exist.
```
Comment (ishg-153) [inline content/css/concepts/overflow/terms/overflow-y/overflow-y.md:53]:
```suggestion
Provide a scrollbar only if overflow content exists.
```
This sentence makes more sense to me.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Jamesllllllllll One small change. Everything else looks good!

**Review by ishg-153 — APPROVED** [review]
@Jamesllllllllll Good to merge!!

**Pull request merged**